<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-color: qlineargradient(spread:reflect, x1:0.0113636, y1:0.028, x2:0.404, y2:0.396864, stop:0 rgba(164, 236, 255, 255), stop:0.869318 rgba(255, 255, 255, 255));
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets.qrc">:/icons/assets/icons/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameAccBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>791</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_update">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>141</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:white;
border: 2px solid rgb(170, 255, 255);
}
QPushButton:hover{
background-color:  rgba(75, 226, 226,0.6);
}</string>
    </property>
    <property name="text">
     <string>Cập nhật</string>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/icons/assets/icons/refresh.svg</normaloff>:/icons/assets/icons/refresh.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_them">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>141</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:white;
border: 2px solid rgb(170, 255, 255);
}
QPushButton:hover{
background-color:  rgba(75, 226, 226,0.6);
}</string>
    </property>
    <property name="text">
     <string>Thêm</string>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/icons/assets/icons/plus.svg</normaloff>:/icons/assets/icons/plus.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_xoa">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>141</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:white;
border: 2px solid rgb(170, 255, 255);
}
QPushButton:hover{
background-color:  rgba(75, 226, 226,0.6);
}</string>
    </property>
    <property name="text">
     <string>Xóa  </string>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/icons/assets/icons/delete.svg</normaloff>:/icons/assets/icons/delete.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dong">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>141</width>
      <height>55</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:white;
border: 2px solid rgb(170, 255, 255);
}
QPushButton:hover{
background-color:  rgba(75, 226, 226,0.6);
}</string>
    </property>
    <property name="text">
     <string>Đóng  </string>
    </property>
    <property name="icon">
     <iconset resource="assets.qrc">
      <normaloff>:/icons/assets/icons/cancel.svg</normaloff>:/icons/assets/icons/cancel.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="table_danhsach_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>911</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>47</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>28</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <column>
    <property name="text">
     <string>STT</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Họ tên</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Khoa/phòng</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Điện thoại</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Email</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tên đăng nhập</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mật khẩu</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>DANH SÁCH NGƯỜI DÙNG</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
